src/boot/boot.ml

```diff
@@ -211,27 +219,49 @@ let testprog lst =
 
 (* Run program *)
 let runprog name lst =
+    (* TODO prog_argv is never used anywhere *)
```

Comment:
Looking at the code in `mexpr.ml` it appears to grab `argv` separately, maybe this is left from some earlier thought on how it should be done?

(not particularly relevant to this PR, but maybe worth considering later)